src/App/App.js

```diff
@@ -53,22 +55,34 @@ function App({ dexieCurrentUserInstance }) {
     Object.entries(projectsWithSyncErrors).forEach((projectWithSyncErrorsEntry) => {
       const projectId = projectWithSyncErrorsEntry[0]
       const { name: projectName, apiDataTablesThatRejectedSyncing } = projectWithSyncErrorsEntry[1]
-
-      const syncErrorUserMessaging = (
-        <div data-testid={`sync-error-for-project-${projectId}`}>
-          <P>{language.error.getPushSyncErrorMessage(projectName)}</P>
-          {language.error.pushSyncErrorMessageUnsavedData}
-          <ul>
-            {apiDataTablesThatRejectedSyncing?.map((rejectedDataTableName) => (
-              <li key={rejectedDataTableName}>
-                {language.apiDataTableNames[rejectedDataTableName]}
-              </li>
-            ))}
-          </ul>
-        </div>
-      )
-
-      toast.error(...getToastArguments(syncErrorUserMessaging))
+      //const currentUrlPath = window.location.href
+      // the test environment wont see client side routing paths (I cosoled it and looked at the console output of the test)
+      // console.log(currentUrlPath)
+      console.log(location.pathname, projectId, location.pathname.includes(projectId))
+
+      //const isErrorSpecificToProject = location.pathname.includes(projectId)
+      // the projectId here isnt necessarily the same one as in the url since we have offline ready projects which will be pushed/synced no matter what project page you are on.
```

Comment:
excellent point

Reply:
isn't it ok if they're pushed/synced though if you're on a different project page? as long as you're only seeing the relevant one to the project page you're currently on? @mmnoo